src/index.ts

```diff
@@ -39,6 +39,8 @@ const response = new plugin.CodeGeneratorResponse({
 
 const options = op.parse(request.parameter);
 
+type.initialize(options);
```

Comment:
this is a red flag. see my comments on type.ts

Reply:
Resolved as per the other thread (adding compilation context in a future PR)